metricbeat/README.md

```diff
@@ -177,16 +178,22 @@ Give your metrics some time to get from your system to ours, and then open [Logz
 | `hostPathRoot` | Fully-qualified [hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath) that will be used to persist Metricbeat registry data. | `/var/lib` |
 | `logzioCert` | Logzio public SSL certificate. | See [values.yaml](https://github.com/logzio/logzio-helm/blob/master/metricbeat/values.yaml). |
 | `metricbeatConfig` | Metricbeat configuration. | See [values.yaml](https://github.com/logzio/logzio-helm/blob/master/metricbeat/values.yaml). |
+| `daemonset.leanConfig` | When set to `true`, sets the Daemonset's Metricbeat modules configuration to a lean configuration of modules. | See [values.yaml](https://github.com/logzio/logzio-helm/blob/master/metricbeat/values.yaml). |
```

Comment:
please add that the lean configuration is the minimal metrics required to populate our dashboards

Reply:
Done.